atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -80,6 +81,10 @@ def run(self):
 
             image = self._ensure_image_registry(image)
 
+            organization = get_registries_organization(self.workflow)
+            if organization:
+                image.enclose(organization)
```

Comment:
This will change the ImageName object in the keys of `self.workflow.builder.parent_images`. I think that's what we want.

Reply:
You *do* think that's what we want, or you don't?

Reply:
I agree, the second one is better.